cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -2213,6 +2213,14 @@ if(ARROW_WITH_GRPC)
   elseif(gRPC_SOURCE STREQUAL "BUNDLED")
     build_grpc()
   elseif(gRPC_SOURCE STREQUAL "SYSTEM")
+    if(BREW_BIN AND NOT gRPC_ROOT)
+      execute_process(COMMAND ${BREW_BIN} --prefix "grpc"
+                      OUTPUT_VARIABLE gRPC_BREW_PREFIX
+                      OUTPUT_STRIP_TRAILING_WHITESPACE)
+      if(gRPC_BREW_PREFIX)
+        set(gRPC_ROOT ${gRPC_BREW_PREFIX})
+      endif()
+    endif()
```

Comment:
Could you do this before `if(ARROW_WITH_GRPC)` to work this with both `AUTO` and `SYSTEM`?

Reply:
Is this really needed?
The grpc formula isn't keg-only. So we can find gRPC without this.
We can find gRPC in GLib CI: https://travis-ci.org/apache/arrow/jobs/548595893#L438-L439

Reply:
OK, I will remove and see if the builds work